src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -81,9 +83,37 @@ internal class RazorCompletionEndpoint : ICompletionHandler, ICompletionResolveH
             _logger = loggerFactory.CreateLogger<RazorCompletionEndpoint>();
         }
 
-        public void SetCapability(CompletionCapability capability)
+        public bool CanResolve(CompletionItem completionItem)
```

Comment:
I see all sorts of changes here but i'm not sure if things actually changed. Did it?

Reply:
I reverted all those, should be easier to read. Expect a PR in the future where I just release a formatter on the whole repo, I hate organizing class members by hand so I use a formatter, but that makes my PRs look terrible if there were pre-existing formatting violations in that file.

Reply:
That sounds awesome!

Reply:
You say that, but I just did a test-balloon and it has modified basically every file I've let it check, so it likely means touching almost every file.

Reply:
Assuming we agree with all of the re-orderings that doesn't sound awful if it's isolated into a single commit.